src/reducers/ui.js

```diff
@@ -1,14 +1,23 @@
 import Immutable from 'immutable';
-import {updateVerticalFlex} from '../util/resize';
+import {
+  updateEditorColumnFlex,
+  updateWorkspaceRowFlex,
+} from '../util/resize';
 
-export const DEFAULT_VERTICAL_FLEX = new Immutable.List(['1', '1', '1']);
+
+const DEFAULT_FLEX = new Immutable.List(['1', '1', '1']);
+export const DEFAULT_WORKSPACE_FLEX = new Immutable.Map({
+  columnFlex: DEFAULT_FLEX,
+  rowFlex: DEFAULT_FLEX,
+  isMaskingOutput: false,
```

Comment:
Hmm yeah. I think you actually suggested this in a comment I somehow dropped; this should be `DEFAULT_WORKSPACE`.